README.md

```diff
@@ -30,17 +30,27 @@ After installing, you need to require Composer's autoloader:
 require 'vendor/autoload.php';
 ```
 
-First you need to get a fresh OAuth2 access token:
+First you need to instantiate the Client with an OAuthAuthentication
 
-> See the [Authentication](#authentication) part below to see how.
+```php
+use Creads\Partners\Client;
+use Creads\Partners\OAuthAccessToken;
+
+$authentication = new OAuthAuthenticationToken('CLIENT_ID', 'CLIENT_SECRET');
+$client = new Client($authentication, []);
```

Comment:
better use `$client = new Client($authentication);` if not parameters are passed